DEPLOYMENT_CHECKLIST.md

```diff
@@ -68,8 +68,9 @@ To deploy run `yarn deploy --network NETWORK` in `packages/contracts` and replac
 
 ### Packages
 
-- [ ] Publish a new version of `@aragon/osx` (`./packages/contracts`) to NPM
-- [ ] Publish a new version of `@aragon/osx-ethers` (`./packages/contracts`) to NPM
+- [ ] Publish a new version of `@aragon/osx-artifacts` (`./packages/contracts`) to NPM
+- [ ] Publish a new version of `@aragon/osx-contracts` (`./packages/contracts/src`) to NPM
```

Comment:
Make sure to notify people affected by this change (e.g., @juliettech13) https://discord.com/channels/672466989217873929/1077542094224236594/1078239707357523988 to avoid confusion about our package names to the outside world.

Reply:
Done :+1: